apps/condo/domains/organization/schema/OrganizationEmployeeRole.js

```diff
@@ -17,6 +20,23 @@ const OrganizationEmployeeRole = new GQLListSchema('OrganizationEmployeeRole', {
             isRequired: true,
         },
 
+        statusTransitions: {
+            schemaDoc: 'Employee status transitions map',
+            type: Json,
+            required: true,
+            defaultValue: DEFAULT_STATUS_TRANSITIONS,
```

Comment:
I think the default value depends on the language. And we should create it in RegisterNewOrganization

For example here: https://github.com/open-condo-software/condo/blob/18f4689e33c79ebaa8aa0a2922d994f594b46609/apps/condo/utils/serverSchema/Organization.js#L32 we create a User Admin Role depends on the language.

I think we can put some logic here: https://github.com/open-condo-software/condo/blob/18f4689e33c79ebaa8aa0a2922d994f594b46609/apps/condo/utils/serverSchema/Organization.js#L5 and chose the right transition graph (depends on language)

Reply:
https://doma.atlassian.net/browse/SBERDOMA-118